### ISSUE TITLE
Sentry alerts not working

### ISSUE THREAD

**Issue by MatheusGalhani** [IanMitchell/sentrydiscord.dev#24]
I added the webhook in the Sentry settings, but the messages are only triggered when I click "test plugin". When I simulate an error in my application, the problem appears in sentry, but it does not appear in my discord channel.

![image](https://user-images.githubusercontent.com/18535441/175828110-6b13d220-c7e7-42d5-80a8-7305a7ef7331.png)

![image](https://user-images.githubusercontent.com/18535441/175828132-8ed58f3a-3fe0-46e1-b21a-2a17a15e3218.png)

![image](https://user-images.githubusercontent.com/18535441/175828161-2601e8e7-72fb-4a81-9458-aa71b51a0a56.png)

Please, can you help me?

**Comment by thewilloftheshadow** [comment]
Do you have this enabled

![image](https://user-images.githubusercontent.com/35580099/175834001-09aa6a59-e7c4-4b4a-8610-edc970747e7d.png)

**Comment by MatheusGalhani** [comment]
My alert config is 

![image](https://user-images.githubusercontent.com/18535441/175835925-5da71aa1-b597-44ec-9f0e-70a02383f054.png)

**Comment by MatheusGalhani** [comment]
@thewilloftheshadow I already test this ownership, but don't work too

![image](https://user-images.githubusercontent.com/18535441/175837651-2b43ea6a-e133-4937-aed3-0122835519de.png)

**Comment by IanMitchell** [comment]
Sorry, been a really long week - still planning on getting to these soon, hopefully Sunday

**Comment by MatheusGalhani** [comment]
@IanMitchell Okay, I'll wait. I'm trying to find what is incorrect.

**Comment by IanMitchell** [comment]
Hm, it does look like you've configured it correctly. Would you mind emailing me your webhook URL? I can turn on debug mode and see if I'm receiving events and just not posting them, or if I'm not getting them at all

**Comment by MatheusGalhani** [comment]
@IanMitchell Yes sure. Is your email [ian.mitchell@hey.com](mailto:ian.mitchell@hey.com)?

**Comment by IanMitchell** [comment]
Got your email, think I figured out the issue and have pushed a change

**Comment by IanMitchell** [comment]
Ah, weird. Mostly just said it was an issue with the webhook payload putting a ton of information into a weird field - I added a length cap that should fix the issue. You should see error reports coming through now!

**Comment by MatheusGalhani** [comment]
@IanMitchell Are there any tweaks I might be trying to make to the sentry?

**Comment by IanMitchell** [comment]
I'm not sure what's happening honestly - I'm not seeing webhook failure alerts get posted, and as far as I can tell y'all have the alert configured correctly. Been a little swamped this month with work and weddings, but I'm trying to dig in when I can. Just to make sure, the errors that aren't getting posted are all new errors right? I think repeat events don't generally send

**Comment by IanMitchell** [comment]
Gotcha. Are you able to capture a sample error payload I can test? You can post it here or email it to me, whichever you prefer! I can run it manually locally and try and figure out what's going wrong

**Comment by MatheusGalhani** [comment]
@IanMitchell Yes, I am forcing new errors and also old errors to test.

**Comment by MatheusGalhani** [comment]
@IanMitchell I was able to use the webhook as follows

![image](https://user-images.githubusercontent.com/18535441/179778495-d1492b46-8139-43c4-a662-4b930076d703.png)


![image](https://user-images.githubusercontent.com/18535441/179778602-49d154ca-69a3-498d-a481-ed5f08011809.png)


The message arrives, but the stack trace does not come. However, it already works for me.

![image](https://user-images.githubusercontent.com/18535441/179778931-947b0669-aa51-42a3-ae44-c58575670f4f.png)

**Comment by IanMitchell** [comment]
Would you both join my Discord server and see if we can debug this together in real time? I think that might help - I'm still stumped on this one, and don't want to leave either of you hanging. https://discord.gg/ian

**Comment by MatheusGalhani** [comment]
@IanMitchell can i close the issue?

**Comment by IanMitchell** [comment]
Sure thing!
